src/ntfsck.c

```diff
@@ -1180,11 +1188,10 @@ static void ntfsck_verify_mft_record(ntfs_volume *vol, s64 mft_num)
 			return;
 		}
 
-		fsck_err_found();
-		ntfs_log_debug("Clear the bit of mft no(%"PRId64") "
+		ntfs_log_error("Clear the bit of mft no(%"PRId64") "
```

Comment:
we don't need to call fsck_err_found() here.

Reply:
this checked with orphaned inode cadidiates which is all inodes execpt already checked inode in tree traverse. So, I confuse it is error or not.
I already modifed that fsck set bitmaps of all inodes which can be opened to disk at first step temporarily. So, clearing bitmap without checking if those bitmap is set or not, is error? because this mismatch can be caused from temporary bitmap set at first step, I decide to remove err_found()